bin/docker/run

```diff
@@ -5,12 +5,36 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 set -e
 
+# CLI handling
+
+pass_credentials_bintray=
+cmd=""
+
+while [ "$1" != "" ]; do
+    param="$1"
+
+    case $param in
+        --credentials-bintray)
+            pass_credentials_bintray=true
+            ;;
+        *)
+            cmd="$@"
+            break
+            ;;
+    esac
+
+    shift
+done
```

Comment:
Yeah, passing them in `beaver bintray upload` should be enough. Just add `BEAVER_DOCKER_VARS="BINTRAY_USER BINTRAY_BIN" $BEAVER_DOCKER_VARS` to the call of `beaver run` here https://github.com/sociomantic-tsunami/beaver/blob/master/bin/bintray/upload#L37 should be enough.